src/ds/ds-private.h

```diff
@@ -86,7 +86,11 @@ namespace librealsense
             ds::RS465_PID,
             ds::RS416_PID,
             ds::RS405_PID,
-            ds::RS455_PID,
+            ds::RS455_PID
+        };
+
+        // ds6 Devices supported by the current version
+        static const std::set<std::uint16_t> ds6_sku_pid = {
```

Comment:
Check if it needs to be placed on ds6-private

Reply:
done